nuxt/components/UI/AppButton.vue

```diff
@@ -8,16 +12,19 @@
 export default {
   props: {
     to: {
-      type: String,
-      required: true
+      default: ''
```

Comment:
Si `to` n'est pas required et qu'on met une valeur par défaut on peut donc créer des link sans url, ce qu'on ne veut pas faire. `required` à remettre